src/components/pages/user/components/delete-account.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react'
+
+import {useViewer} from 'context/viewer-context'
+import {Dialog, Transition} from '@headlessui/react'
```

Comment:
Using a headlessui dialog because we have one functioning well already in the repo (as opposed to refactoring to radixui)